File: app/src/main/java/ro/mdc_software/coinhivesdkandroidexample/fragments/CoinHiveExampleFragment.java
package ro.mdc_software.coinhivesdkandroidexample.fragments;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import ro.mdc_software.coinhivesdkandroidexample.R;

import ro.mdc_software.coinhive.CoinHiveFragment;

/**
 * @author Cosmin Mihu
 */
public class CoinHiveExampleFragment extends CoinHiveFragment {


    public CoinHiveExampleFragment() {
        // Required empty public constructor
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_coinhive_example, container, false);
    }

    @Override
    public boolean isShowMining() {
        return true;
    }
}

File: app/src/main/java/ro/mdc_software/coinhivesdkandroidexample/fragments/CoinHiveExampleFragment.kt
package ro.mdc_software.coinhivesdkandroidexample.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import ro.mdc_software.coinhive.CoinHiveFragment
import ro.mdc_software.coinhivesdkandroidexample.R

/**
 * @author Cosmin Mihu
 */
class CoinHiveExampleFragment : CoinHiveFragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater?.inflate(R.layout.fragment_coinhive_example, container, false)
    }

    override fun isShowMining() = true
}
